docs/current_architecture.md

```diff
@@ -91,4 +91,80 @@ classDef erlang fill:#b83998,color:white;
 classDef php fill:#4f5d95,color:white;
 ```
 
+Follow these links for the current state of [metric](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/metric_service_features.md) and [trace](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/trace_service_features.md) instrumentation of the demo applications.\
+The collector is configured in [otelcol-config.yml](https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/otelcollector/otelcol-config.yml), alternative exporters can be configured here.
+
+```mermaid
+graph TB
+subgraph tdf[Telemetry Data Flow]
+    subgraph subgraph_padding [ ]
+        style subgraph_padding fill:none,stroke:none;
+        %% padding to stop the titles clashing
+        subgraph od[Open Telemetry Demo]
+        ms(Microservice)
+        end
+
+        ms -.->|"OTLP<br/>gRPC"| oc-grpc
+        ms -.->|"OTLP<br/>HTTP POST"| oc-http
+
+        subgraph oc[OTel Collector]
+            style oc fill:#97aef3; 
+            oc-grpc[/"OTLP Receiver<br/>listening on<br/>grpc://localhost:4317/"/]
+            oc-http[/"OTLP Receiver<br/>listening on <br/>http://localhost:4318/<br/>https://localhost:4318/"/]
+            oc-proc(Processors)
+            oc-prom[/"Prometheus Exporter<br/>listening on<br/>http://localhost:9464/"/]
+            oc-jag[/"Jaeger Exporter"/]
+
+            oc-grpc --> oc-proc
+            oc-http --> oc-proc
+
+            oc-proc --> oc-prom
+            oc-proc --> oc-jag
+        end
+
+        oc-prom -->|"http://localhost:9464/metrics"| pr-sc
+        oc-jag -->|gRPC| ja-col
+
+        subgraph pr[Prometheus]
+            style pr fill:#e75128;
+            pr-sc[/"Prometheus Scraper<br/>polling every 5 seconds"/]
+            pr-tsdb[(Prometheus TSDB)]
+            pr-http[/"Prometheus HTTP<br/>listening on<br/>http://localhost:9090"/]
+
+            pr-sc --> pr-tsdb
+            pr-tsdb --> pr-http
+        end
+
+        pr-b{{"Browser<br/>Prometheus UI"}}
+        pr-http ---->|"http://localhost:9090/graph"| pr-b
+    
+        subgraph ja[Jaeger]
+            style ja fill:#60d0e4;
+            ja-col[/"Jaeger Collector<br/>listening on<br/>grpc://jaeger:4317/"/]
+            ja-tsdb[(Jaeger TSDB)]
```

Comment:
Not blocking this PR, might worth double checking if "Jaeger TSDB" is the precise term (I guess it might just be a "DB (database)" instead of "TSDB (timeseries database)").